ansible/roles/stack-sunbird/templates/sunbird_learner-service.env

```diff
@@ -107,3 +107,5 @@ sunbird_subdomain_keycloak_base_url={{sunbird_subdomain_keycloak_base_url | d(''
 #Required for cert-service
 sunbird_cert_domain_url={{proto}}://{{proxy_server_name}}
 sunbird_cert_enc_service_url=enc-service
+#Required by Learner-service to call cert service
+sunbird_cert_service_base_url=http://cert-service:9011
```

Comment:
Hey @anmol2302 the value of the cert service internal variable is wrong, please update this